src/commands/GanacheCommands.ts

```diff
@@ -74,4 +77,9 @@ export namespace GanacheCommands {
     });
     return (pick as LocalProject).port;
   }
+
+  export async function getGanacheProjectOptions(projectView?: ProjectView): Promise<TLocalProjectOptions> {
+    const project: LocalProject = projectView?.extensionItem as LocalProject;
```

Comment:
if extension item is null or projectview in this case wouldn't line 83 cause a null pointer/object undefined error in this case?